lib/src/utils.ts

```diff
@@ -2,4 +2,30 @@
 // MIT-style license that can be found in the LICENSE file or at
 // https://opensource.org/licenses/MIT.
 
+import {URL} from 'url';
+
+import * as proto from './vendor/embedded_sass_pb';
+import {SourceLocation} from './exception/location';
+import {SourceSpan} from './exception/span';
+
 export type PromiseOr<T> = T | Promise<T>;
+
+export function deprotifySourceSpan(span: proto.SourceSpan): SourceSpan {
+  return {
+    text: span.getText(),
+    start: deprotifySourceLocation(span.getStart()!),
```

Comment:
We should explicitly verify that these fields aren't null ("the endpoint that receives [a message] must reject that message if [a mandatory field] is not set") and throw a user-facing error if they are, since it constitutes a protocol violation.